apps/web/components/BackgroundImage/BackgroundImage.tsx

```diff
@@ -14,21 +14,24 @@ export type BackgroundImageProps = {
   boxProps?: BoxProps
   positionX?: 'left' | 'right'
   backgroundSize?: 'cover' | 'contain'
+  useThumbnail?: boolean
 }
 
-const useImageLoader = (url: string): boolean => {
+const useImageLoader = (url: string, shouldLoad?: boolean): boolean => {
   const isMounted = useMountedState()
   const [loaded, setLoaded] = useState(false)
 
   useEffect(() => {
-    const img = new window.Image(100)
-    img.onload = img.onerror = () => {
-      if (isMounted) {
-        setLoaded(true)
+    if (shouldLoad) {
+      const img = new window.Image(100)
+      img.onload = img.onerror = () => {
+        if (isMounted) {
```

Comment:
Not really related to this change in particular, but..

I'm not sure this mounted check would work and/or be accurate if this fires post-mounting.. since the scope of this function will still be seeing the older version of isMounted.. it would be much better if you'd return a function to simply unsubscribe from this event (unassign the error handlers)